dojo/urls.py

```diff
@@ -152,17 +150,7 @@
     url(r'^schema/$', SchemaView.as_view(), name='schema'),
 ]
 
-schema_view = get_schema_view(
-    openapi.Info(
-        title="Defect Dojo API",
-        default_version='v2',
-        description="To use the API you need be authorized.",
-    ),
-    # if public=False, includes only endpoints the current user has access to
-    public=True,
-    # The API of a OpenSource project should be public accessible
-    permission_classes=[permissions.AllowAny],
-)
+schema_view = get_swagger_view(title='Defect Dojo API v2')
```

Comment:
You are undoing #1664 here. Please pull recent changes.

Reply:
Fixed through later rebase. Marking as resolved.